### PR TITLE
feat(17757) - Sanitized Logging

### PR DESCRIPTION
**Pull request by simon622**
Analysed Logging with Standard Adapers Configured. Fixed PLC4X logging level in log config. Ensured HiveMQ log lines were appropriately formatted and reflected at the correct levels.Fixed some incorrect logger use. Applied log level conditionalcheck for high volume code paths.

### CODE REVIEW

File: hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java [hunk 1]
```diff
@@ -239,10 +235,12 @@ public Response updateAdapter(final String adapterId, final Adapter adapter) {
     @Override
     public Response deleteAdapter(final String adapterId) {
         Optional<ProtocolAdapterWrapper> instance = protocolAdapterManager.getAdapterById(adapterId);
-        if (!instance.isPresent()) {
+        if (instance.isEmpty()) {
             return ApiErrorUtils.notFound("Adapter not found");
         }
-        logger.info("deleting adapter {}", adapterId);
+        if(logger.isTraceEnabled()){
```
Comment (DC2-DanielKrueger) [inline hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java:241]: I would suggest debug as log level for the operations. They wont be called that often and they might be useful for debugging, wdyt?

File: hivemq-edge/src/main/java/com/hivemq/protocols/ProtocolAdapterManager.java [hunk 2]
```diff
@@ -235,12 +244,12 @@ public CompletableFuture<Void> stop(final @NotNull ProtocolAdapter protocolAdapt
...
         stopFuture.thenApply(input -> {
             if(log.isTraceEnabled()){
-                log.trace("Protocol-adapter {} stopped", protocolAdapter.getId());
+                log.trace("Protocol-Adapter \"{}\" Stopped", protocolAdapter.getId());
             }
             return null;
         }).exceptionally(throwable -> {
             if(log.isWarnEnabled()){
-                log.warn("Protocol-adapter {} was unable to stop cleanly",
+                log.warn("Protocol-Adapter \"{}\" Was Unable To Stop Cleanly",
```
Comment (DC2-DanielKrueger) [inline hivemq-edge/src/main/java/com/hivemq/protocols/ProtocolAdapterManager.java:252]: why are the words starting with a capital letter?

**Comment by simon622** [comment]
Log messages reverted to sentence capitalisation

File: hivemq-edge/src/main/java/com/hivemq/protocols/ProtocolAdapterManager.java [hunk 3]
```diff
@@ -297,7 +306,7 @@ public boolean deleteAdapter(final String id) {
                 } finally {
                     eventService.fireEvent(
                             eventBuilder(Event.SEVERITY.WARN, adapterInstance.get().getAdapter()).
-                            withMessage(String.format("Adapter '%s' was deleted from the system permanently",
+                            withMessage(String.format("Adapter \"%s\" Was Deleted From The System Permanently",
```
Comment (DC2-DanielKrueger) [inline hivemq-edge/src/main/java/com/hivemq/protocols/ProtocolAdapterManager.java:309]: Please revert the formatting.

Comment (DC2-DanielKrueger) [reply hivemq-edge/src/main/java/com/hivemq/protocols/ProtocolAdapterManager.java:309]:
```suggestion
                            withMessage(String.format("Adapter \"%s\" was deleted from the system permanently.",
```

**Pull request merged**